modules/serverless-sinkbinding-reference.adoc

```diff
@@ -204,3 +204,12 @@ This sets the `K_CE_OVERRIDES` environment variable on the `subject`:
 ----
 { "extensions": { "extra": "this is an extra attribute", "additional": "42" } }
 ----
+
+== The include label
```

Comment:
```suggestion
== Include label
```

I don't think you need "The" here. Looking at the ones above it seems that convention is to specify the item then type of item so in this case, "Include label" or above case "Subject parameter".